activemodel/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   Encapsulate each validation error as an Error object.
 
+    The `ActiveModel`’s `errors` collection is now an array of these Error 
+    objects, instead of messages/details hash.
+
+    For each of these `Error` object, its `message` and `full_message` methods 
+    are for generating error messages. Its `details` method would return error’s 
+    extra parameters, found in the original `details` hash.
+
+    The change tries its best at maintaining backward compatibility, however 
```

Comment:
@lulalala : I think you might want to add methods which were deprecated in this change into changelog too. I have seen that pattern but I am not so sure. Maybe Rails core team member could comment on it.

Reply:
Also, I would skip this paragraph and add some examples for before/after the change. Something like this: https://github.com/rails/rails/blame/6-0-stable/activemodel/CHANGELOG.md#L40-L51

But before you do the change get it confirmed from the team member as well. They might have a different opinion about it.